### ISSUE TITLE
Automount CD/ISO

### ISSUE THREAD

**Issue by ToniBC** [libretro/dosbox-svn#21]
Hello, I wanted to comment this because although it is not exactly a mistake, it would be nice to implement it.

I do not know if it will be, but although an option was incorporated to mount the CD with a .iso or .cue, this process has to be done manually and ideally it would be automatically.

This I think is a mistake, because the games that require CD do not work. As I repeat, there may be an option to do it, but I can't find anything in the documentation.

Ideally, include a file like .m3u or something similar, that if you put it in the game directory, load the iso that is marked there, or when the iso is loaded by the Retroarch menu, you can save that configuration.

If I create a .conf file, if you read it you ignore the internal configuration of the core (the retroarch options), which also complicates everything.

Is there any solution for this?

**Comment by realnc** [comment]
> If I create a .conf file, if you read it you ignore the internal configuration of the core

That's not true. I use a conf file (for autoexec) and the core options work fine.

**Comment by ToniBC** [comment]
Ok, I see it, just putting [autoexec] would just load that and it doesn't interfere with the other options. Bracias

I get to load the iso that way, but the iso doesn't appear as loaded in the Retroarch menu and if any game has more than 1 CD, the load has to be manual.

The idea would be to load 1 or several isos automatically, such as when loading a m3u file in Retroarch.

**Comment by fr500** [comment]
I'll think of something

**Comment by 3XistencE** [comment]
Can be created an option inside the core options menu where people can select one iso to pre-mount.
set as variable to load into the autoexec (if empty/default will be skipped) otherwise it will try to mount the cd image on load.
I must observe that it can be also useful to set a Main/mother mount point directly from the core options, some software require a specific path to be respected and the main executable can't exist in c:/

**Comment by mediamoshpit** [comment]
Relating to this issue, does dosbox-svn allow swapping multiple discs loaded from a .conf file using DOSBox's native ctrl+f4 command? I ask because I'm about to start a game where I'll need to invest a lot of time before the situation comes up and I'd like to avoid the potential problem.

**Comment by realnc** [comment]
> Relating to this issue, does dosbox-svn allow swapping multiple discs loaded from a .conf file using DOSBox's native ctrl+f4 command? I ask because I'm about to start a game where I'll need to invest a lot of time before the situation comes up and I'd like to avoid the potential problem.

Nope. But I believe dosbox-pure added m3u8 support which populates the RetroArch disc image menu with the images from the m3u8:

https://github.com/libretro/dosbox-pure#loading-m3u8-files

**Comment by mediamoshpit** [comment]
Ah, very kool! This particular setup is using Lakka and I don't  think Pure is an option there yet. I actually managed to get it to work in SVN by mounting the second disc as drive E. When the game needs the other disc I guess it queries the other available drives. This is probably game specific, but it might help others. Thank you much for the reply.
